nhost/compose/config.go

```diff
@@ -315,6 +315,13 @@ func (c Config) functionsServiceEnvs() env {
 	e := env{}
 	e.merge(env{
 		"NHOST_BACKEND_URL":    c.envValueNhostBackendUrl(),
+		"NHOST_SUBDOMAIN":      "localhost",
+		"NHOST_REGION":         "",
+		"NHOST_HASURA_URL":     fmt.Sprintf("http://localhost:%d", proxyPort),
+		"NHOST_GRAPHQL_URL":    fmt.Sprintf("http://localhost:%d/v1/graphql", proxyPort),
+		"NHOST_AUTH_URL":       fmt.Sprintf("http://localhost:%d/v1/auth", proxyPort),
+		"NHOST_STORAGE_URL":    fmt.Sprintf("http://localhost:%d/v1/storage", proxyPort),
+		"NHOST_FUNCTIONS_URL":  fmt.Sprintf("http://localhost:%d/v1/functions", proxyPort),
```

Comment:
If these ENVs are used inside `functions` container, they won't work because of networking

Reply:
will fix :+1: